addons/nuxeo-drive-server/nuxeo-drive-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-drive/nuxeo-drive-desktop-packages.html

```diff
@@ -125,26 +125,27 @@
           var prefix = 'nuxeo-drive';
           var baseUrl = 'https://community.nuxeo.com/static/drive-updates';
 
-          var name = prefix + '.dmg';
+          var name = prefix + '-x86_64.AppImage';
           pkgs.push({
             name: name,
-            platform: 'osx',
+            platform: 'GNU/Linux',
```

Comment:
The name that everyone uses is `Linux`. I don't think this is the place for GNU evangelism.

Reply:
This is not GNU evangelism but correctness. Also folks using Linux know that the full qualification is GNU/Linux, they will not be lost.
This is not something weird I am changing, I just want to be correct and on the same line as Drive as well (where there is not occurrence of "Linux").

But I do not mind removing the "GNU" part if you insist :)

Reply:
I'm well aware of the arguments and I still call it evangelism, sorry. Let's discuss it next time we cross path if you prefer.

Reply:
:+1: and I will push a commit to remove the "GNU/" part.